app/controllers/insights_cloud/tasks_controller.rb

```diff
@@ -2,7 +2,12 @@ module InsightsCloud
   class TasksController < ::ApplicationController
     def create
       selected_org = Organization.current
-      InsightsCloud::Async::InsightsFullSync.perform_now(selected_org)
+      host_statuses = InsightsCloud::Async::InsightsFullSync.perform_now(selected_org)
+
+      render json: {
```

Comment:
@adamruzicka hey, looks like it doesn't get to line 7 after the task was finished,
any idea how can I invoke it based on the [tasks returned data](https://github.com/theforeman/foreman_rh_cloud/pull/276/commits/d0126586a493f7cfecff96c930158807ede2ef7a#diff-0adbd32f5f95abd3341cafdf4938d09bR33)?

Reply:
If I'm reading https://github.com/theforeman/foreman_rh_cloud/pull/276/files#diff-306836a7c92f19b1dd2807d8d47f1aefR3 and https://github.com/theforeman/foreman_rh_cloud/pull/276/files#diff-21497849d8f00507c9c8dcaf6288b136R17-R19 right, then you're hitting this controller https://github.com/theforeman/foreman_rh_cloud/pull/276/files#diff-8ed246688f8a35bd543d23989dbdd4eaR2 instead of this one

Reply:
oops, good catch!! will try that soon :)

Reply:
awesome! it worked !